superset/views/dashboard/api.py

```diff
@@ -169,6 +173,7 @@ class DashboardRestApi(DashboardMixin, BaseSupersetModelRestApi):
     method_permission_name = {
         "get_list": "list",
         "get": "show",
+        "export": "list",
```

Comment:
Switched export to a `list` permission to reduce the amount of permissions

Reply:
👍 makes sense as it's essentially another form of list

Reply:
Seems like a different level of permission if the export also contains all of the chart and data source info. Was this the `can_download_dashboards` permission? Can we use the same?

Reply:
On this case is `mulexport` changed it